### PR TITLE
[NUI] Add ExcludeLayouting to View

### PR DESCRIPTION
**Pull request by bshsqa**
- If ExcludeLayouting property is false, the view is not affected by parent layout
 - Even the property is false, the children Views are affected by layout of current View
 - This ExcludeLayouting property is changed synchronously with SetPositionByLayout of its layout
 - Children of AbsoluteLayout can freely be animated.

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 1]
```diff
@@ -205,6 +206,24 @@ protected set
             }
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool TransformedByLayout
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:210]: Please leave a description comment in the case of hidden API.

**Comment by bshsqa** [comment]
Currently on changing property name.
Please do not merge this.

File: src/Tizen.NUI/src/public/BaseComponents/View.cs (hunk 1 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs]: done. thank you

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by hyunjushin — APPROVED** [review]

**Pull request merged**